From b1761565882ed6a171136c2c89e597bc4dd5b6bf Mon Sep 17 00:00:00 2001
From: Richard Biener <rguenther@suse.de>
Date: Wed, 1 Aug 2018 08:06:20 +0000
Subject: [PATCH] re PR tree-optimization/86724 (Compilation error with new isl
 0.20 (missing includes))

2018-08-01  Richard Biener  <rguenther@suse.de>

	PR bootstrap/86724
	* graphite.h: Include isl/id.h and isl/space.h to allow build
	with ISL 0.20.

From-SVN: r263194
---
 gcc/ChangeLog  | 6 ++++++
 gcc/graphite.h | 2 ++
 2 files changed, 8 insertions(+)

diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index 6f30129db98..a1c08b7831b 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,9 @@
+2018-08-01  Richard Biener  <rguenther@suse.de>
+
+	PR bootstrap/86724
+	* graphite.h: Include isl/id.h and isl/space.h to allow build
+	with ISL 0.20.
+
 2018-08-01 Jan Willem Jagersma  <jwjagersma@gmail.com>
 
 	PR target/86651
diff --git a/gcc/graphite.h b/gcc/graphite.h
index 4e0e58c60ab..be0a22b3894 100644
--- a/gcc/graphite.h
+++ b/gcc/graphite.h
@@ -37,6 +37,8 @@ along with GCC; see the file COPYING3.  If not see
 #include <isl/schedule.h>
 #include <isl/ast_build.h>
 #include <isl/schedule_node.h>
+#include <isl/id.h>
+#include <isl/space.h>
 
 typedef struct poly_dr *poly_dr_p;
 
-- 
2.31.1

